distribution/docs/src/main/jdocs/content/_security-framework/security-expansion-service-contents.adoc

```diff
@@ -0,0 +1,72 @@
+:title: Expansion Service
+:type: securityFramework
+:status: published
+:parent: Security Framework
+:children: na
+:order: 08
+:summary: Expansion Service.
+
+The Expansion Service and its corresponding expansion-related commands provide an easy way for developers to add expansion capabilities to ${branding} during user attribute and metadata card processing.
+In addition to these two defined uses of the expansion service, developers are free to utilize the service in their own implementations.
+
+Each instance of the expansion service consists of a collection of rulesets.
+Each ruleset consists of a key value and its associated set of rules.
+Callers of the expansion service provide a key and an original value to be expanded.
```

Comment:
I think it reads better without the word "original".

Reply:
agreed. updated.